### PR TITLE
Ignore auth in dev environment

### PR DESCRIPTION
**Pull request by kelvin-chappell**
/cc @paulmr

### CODE REVIEW

File: app/di.scala [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 import com.gu.atom.publish.AtomPublisher
 import com.gu.pandomainauth.action.AuthActions
 import data.AtomPublisherProvider
 
 class Module extends AbstractModule {
   def configure() = {
-    bind(classOf[AuthActions])
-    .to(classOf[controllers.PanDomainAuthActions])
+    
+    if (currentStage() == DEVELOPMENT) {
```
Comment (paulmr) [inline app/di.scala]: do we need to do anything to make sure that this is `PRODUCTION` on CODE?

**Comment by paulmr** [comment]
as long as this doesn't disable panda on CODE, :+1:

File: app/di.scala (hunk 1 above)
Comment (kelvin-chappell) [inline app/di.scala]: I'll check.

**Comment by jamesgorrie** [comment]
Why is this a good idea? Aren't we opening our selves to breaking things in PROD that we won't catch in Dev?

**Comment by kelvin-chappell** [comment]
@jamesgorrie I think you might be right.  There's a greater risk of building unsecured endpoints etc with this change.  We should keep dev as similar to prod as possible.  I'll take @paulmr's advice and get access to to the S3 bucket instead.

**Pull request closed without merging**